OMF_APITests/UnitTest1.cs

```diff
@@ -1,14 +1,249 @@
+
+            bool success = true;
+
+            foreach (var endpoint in endpoints)
+            {
+                try
+                {
+                    // delete containers
+                    foreach (var omfContainer in omfContainers)
+                    {
+                        string omfContainerString = $"[{JsonConvert.SerializeObject(omfContainer)}]";
+                        OMF_API.Program.sendMessageToOmfEndpoint(endpoint, "container", omfContainerString, "delete");
+                    }
+
+                    // delete types
+                    foreach (var omfType in omfTypes)
+                    {
+                        string omfTypeString = $"[{JsonConvert.SerializeObject(omfType)}]";
+                        OMF_API.Program.sendMessageToOmfEndpoint(endpoint, "type", omfTypeString, "delete");
+                    }
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine($"Encountered Error: {e}");
+                    success = false;
+                    throw;
+                }
+            }
+
+            return success;
+        }
+
+        private bool compareData(dynamic response, dynamic sentData)
+        {
+            bool success = true;
+
+            foreach (JProperty property in sentData["values"][0])
+            {
+                if (property.Name != "Timestamp" && ((string)property.Value != (string)response.Property(property.Name).Value))
+                    success = false;
+            }
+
+            return success;
+        }
+
+        private bool compareData(string itemName, dynamic response, dynamic sentData)
+        {
+            bool success = true;
+
+            var split = itemName.Split(".");
+            if (split.Length == 2)
+            {
+                string key = split[1];
+                foreach (JProperty property in sentData["values"][0])
+                {
+                    if (key == property.Name && ((string)property.Value != (string)response))
+                        success = false;
+                }
+            }
+            else
+            {
+                string key = split[0];
+                foreach (JProperty property in sentData["values"][0])
+                {
+                    if (property.Name != "Timestamp" && ((string)property.Value != (string)response))
+                        success = false;
+                }
+            }
+
+            return success;
+        }
+
+        private static async Task<HttpResponseMessage> sendGetRequestToEndpoint(Endpoint endpoint, string uri)
+        {
+            // create a request
+            HttpRequestMessage request = new HttpRequestMessage()
+            {
+                Method = HttpMethod.Get,
+                RequestUri = new Uri(uri)
+            };
+
+            // add headers to request
+            request.Headers.Add("Accept-Verbosity", "verbose");
+            if (string.Equals(endpoint.EndpointType, "OCS"))
```

Comment:
Use the `string.Equals()` overload that specifies a case insensitive `StringComparisonType` here and line 237.

Reply:
Looks like you missed the `StringComparisonType` here